File: challenges/hacker/026/EpicJhon.js
// http://www.hacker.org/challenge/chal.php?id=26

var x = 2;
var y = 5;
var z = 3;
var a = new Array(6, 5, 4, 9, 3, 7);
var v = y * a[y] + z * a[z] - y * x * z + a[y] * a[x] * a[3];
a.reverse;
v = v + a[z] + a[y] + 11 * a[1] + a[0] - 1;
console.debug(v);

File: challenges/hacker/015/EpicJhon.java
/*http://www.hacker.org/challenge/chal.php?id=15*/


/* package whatever; // don't place package name! */

import java.util.*;
import java.lang.*;
import java.io.*;

/* Name of the class has to be "Main" only if the class is public. */
class Ideone
{
	
	public static int bucketFromRandom(int randomNumber) {
	int a[]	= new int[10];
	for (int i = 0; i < a.length; i++)
		a[i] = i * randomNumber;
	int index = Math.abs(randomNumber) % a.length;
	return a[index];
}

	public static void main(String[] args) {
		for (int i = Integer.MIN_VALUE; i <= Integer.MAX_VALUE; i++) {
			try{
		bucketFromRandom(i);	
			} catch(Exception e) {
		System.out.println(i);			
			}
		}
		
	}
}
